### PR TITLE
Improve documentation of `lazy` property on TabNavigatorConfig

### PR DESCRIPTION
**Pull request by rgilbert1**
I added my changes to `bottom-tab-navigator` in the version 3.x docs, I hope that was the right thing to do 😬

### CODE REVIEW

File: website/versioned_docs/version-3.x/tab-navigator.md [hunk 1]
```diff
@@ -0,0 +1,126 @@
+---
+id: version-3.x-tab-navigator
+title: createTabNavigator
+sidebar_label: createTabNavigator
```
Comment (brentvatne) [inline website/versioned_docs/version-3.x/tab-navigator.md]: createTabNavigator is gone in react-navigation@>=3.0, we have `createBottomTabNavigator` and `createMaterialTopNavigator` instead now :) so we can remove this doc

**Comment by brentvatne** [comment]
aside from that one comment i left it looks good to me! let me know when that change is ready and i'll land it

File: website/versioned_docs/version-3.x/tab-navigator.md (hunk 1 above)
Comment (rgilbert1) [reply website/versioned_docs/version-3.x/tab-navigator.md]: Got it, makes sense that the file wasn't there already then!

**Comment by rgilbert1** [comment]
Thanks Brent! We should be good to go!

**Comment by brentvatne** [comment]
thank you @rgilbert1! 😄

**Pull request merged**